src/assets/CurrencyRateController.ts

```diff
@@ -128,9 +128,10 @@ export class CurrencyRateController extends BaseController<CurrencyRateConfig, C
 	 */
 	async fetchExchangeRate(currency: string, nativeCurrency = this.activeNativeCurrency): Promise<CurrencyRateState> {
 		const json = await handleFetch(this.getPricingURL(currency, nativeCurrency));
+		const exchange: string | undefined = json[currency.toUpperCase()];
 		return {
 			conversionDate: Date.now() / 1000,
-			conversionRate: Number(json[currency.toUpperCase()]),
+			conversionRate: exchange === undefined ? 0 : Number(exchange),
```

Comment:
Would it make sense to set this to `null` instead, or throw an error? We have to handle errors from this function anyway, since an HTTP 500 response would result in one. Setting it to 0 doesn't seem especially useful :thinking:

Reply:
I could go for throwing an error, that could be more appropriate. I'd rather not propagate `null` values.

Reply:
Sure, fair enough.

Though on second glance, we need to handle `0` already regardless, since it's the default value. So that ship has already sailed. Not sure I love that as a default value, but your original suggestion of leaving it as `0` seems appropriate.